### PR TITLE
Implement Monaco code editor example

### PR DESCRIPTION
**Pull request by foxymiles**
**Public-Facing Changes**
This adds a simple example extension that contains the Monaco code editor.

**Description**
This uses the https://github.com/suren-atoyan/monaco-react library to embed the editor in an extension panel.

Fixes https://github.com/foxglove/create-foxglove-extension/issues/68

### CODE REVIEW

File: examples/monaco-editor-example/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "create-foxglove-extension/tsconfig/tsconfig.json",
```
Comment (defunctzombie) [inline examples/monaco-editor-example/tsconfig.json:2]: Is this extends right? I thought we use foxglove/tsconfig?

**Review by defunctzombie — APPROVED** [review]

File: examples/monaco-editor-example/tsconfig.json (hunk 1 above)
Comment (foxymiles) [reply examples/monaco-editor-example/tsconfig.json:2]: That's what you get out of the box from running:

```bash
npm init foxglove-extension@latest helloworld
```

Do we need to change it?

Comment (defunctzombie) [reply examples/monaco-editor-example/tsconfig.json:2]: Or maybe the idea was to provide a tsconfig specific to extensions authoring. I could see that having been the thinking for this.

Comment (foxymiles) [reply examples/monaco-editor-example/tsconfig.json:2]: Yeah I can imagine that might be useful.

**Pull request merged**